.eslintrc

```diff
@@ -2,5 +2,11 @@
   "extends": [ "apostrophe", "plugin:vue/vue3-recommended" ],
   "globals": {
     "apos": true
+  },
+  "rules": {
+    "no-unused-vars": [
+      "error",
+      { "varsIgnorePattern": "^_.?" }
```

Comment:
I don't think we need this, this seems to have been added for test but it targets everything.
I creating variables for something we don't plan to use, I don't think it's worth it.

```js
const [ _, name ] = regex.exec(fullname);
// below is valid without the extra rule
const [ , name ] = regex.exec(fullname);
```

Reply:
Rule has been removed.